src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -50,8 +58,14 @@ public PersonCard(Person person, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
         phone.setText(person.getPhone().value);
+        Image phone = new Image(getClass().getResourceAsStream("/images/phone_icon.png"));
```

Comment:
Image path name "/images/phone_icon.png" could be made into a final static variable for better organisation.